libs/shared/types/src/lib/types.ts

```diff
@@ -51,6 +113,7 @@ export interface Event {
   status: EventStatus;
   endpoints: EventEndpoint[];
   logs: EventLog[];
+  inputType: InputType;
```

Comment:
Could we move this in the Source object itself instead of having it next to the source?